packages/core-services/src/types/IAbacService.ts

```diff
@@ -8,35 +8,43 @@ import type {
 	ILDAPEntry,
 } from '@rocket.chat/core-typings';
 
+export type AbacActor = Pick<IUser, '_id' | 'username' | 'name'>;
+
 export interface IAbacService {
-	addAbacAttribute(attribute: IAbacAttributeDefinition): Promise<void>;
-	listAbacAttributes(filters?: {
-		key?: string;
-		values?: string;
-		offset?: number;
-		count?: number;
-	}): Promise<{ attributes: IAbacAttribute[]; offset: number; count: number; total: number }>;
-	listAbacRooms(filters?: {
-		offset?: number;
-		count?: number;
-		filter?: string;
-		filterType?: 'all' | 'roomName' | 'attribute' | 'value';
-	}): Promise<{ rooms: IRoom[]; offset: number; count: number; total: number }>;
-	updateAbacAttributeById(_id: string, update: { key?: string; values?: string[] }): Promise<void>;
-	deleteAbacAttributeById(_id: string): Promise<void>;
+	addAbacAttribute(attribute: IAbacAttributeDefinition, actor?: AbacActor): Promise<void>;
```

Comment:
Shouldn't we consider the actor a non-optional parameter (but yet possibly undefined)?